src/traces/sunburst/attributes.js

```diff
@@ -203,6 +203,18 @@ module.exports = {
             'this option won\'t have any effect and `insidetextfont` would be used.'
         ].join(' ')
     }),
+    rotation: {
+        valType: 'number',
+        role: 'style',
+        min: -180,
+        max: 180,
+        dflt: 0,
+        editType: 'plot',
+        description: [
+            'Instead of the first slice starting at 3 o\'clock,',
+            'rotate to some other angle.'
```

Comment:
```suggestion
            'Rotates the whole diagram counterclockwise by some angle.',
            'By default the first slice starts at 3 o\'clock.'
```

Reply:
last commit (f90737b) contains this proposition